great_expectations/dataset/base.py

```diff
@@ -664,101 +663,101 @@ def validate(self, expectations_config=None, catch_exceptions=True, output_forma
 
     ##### Output generation #####
     def _format_column_map_output(self,
-        output_format, success,
-        element_count,
-        nonnull_values, nonnull_count,
-        boolean_mapped_success_values, success_count,
-        exception_list, exception_index_list
-    ):
+                                  result_format, success,
+                                  element_count,
+                                  nonnull_values, nonnull_count,
+                                  boolean_mapped_success_values, success_count,
+                                  unexpected_list, unexpected_index_list
+                                  ):
         """Helper function to construct expectation result objects for column_map_expectations.
 
-        Expectations support four output_formats: BOOLEAN_ONLY, BASIC, SUMMARY, and COMPLETE.
+        Expectations support four result_formats: BOOLEAN_ONLY, BASIC, SUMMARY, and COMPLETE.
         In each case, the object returned has a different set of populated fields.
-        See :ref:`output_format` for more information.
+        See :ref:`result_format` for more information.
 
         This function handles the logic for mapping those fields for column_map_expectations.
         """
-        if output_format == "BOOLEAN_ONLY":
-            return_obj = success
 
-        elif output_format == "BASIC":
-            exception_count = len(exception_list)
+        # Retain support for string-only output formats:
+        if isinstance(result_format, string_types):
```

Comment:
Yikes. So we *are* allowing nested `result_format` objects? I didn't see this in the docs.

Reply:
Discussed...it's there!